### DISCUSSION TITLE
Cannot Run Script HELP!!

### DISCUSSION THREAD

**Discussion by LetsGrow2024** [community/community#119999] [Programming Help]
### Select Topic Area

Question

### Body

I am trying to this script but I keep getting this error, saying it cannot find the file, but two snips down there is the file in the location I dont understand. any help would be huge
![image](https://github.com/community/community/assets/165970590/d5898424-649b-45a1-9837-e05985018bd2)
![image](https://github.com/community/community/assets/165970590/51d7c687-e4d1-497f-8135-42bece169afa)

**Comment by grandmas-favorite** [comment]
Hi @LetsGrow2024!

Correct me if I'm wrong, but it appears that the "woodcutting" program references another path called \.venv\Scripts\python.exe, but I don't see that path here - is that the file you're referencing?

**Reply by LetsGrow2024** [reply]
First love your name. But the second one is my directories so it says running from this directories and that’s the path to where the script is. I honestly don’t know how to change the first part it shows two different directories there and anytime I run a python script it shows both I don’t know why
